lib/ansible/modules/system/puppet.py

```diff
@@ -260,6 +266,8 @@ def main():
             cmd += " --certname='%s'" % p['certname']
         if module.check_mode:
             cmd += " --noop"
+        if p['no_use_srv_records']:
```

Comment:
~~~python
        if p['use_srv_records'] is not None:
            if not p['use_srv_records']:
                cmd += " --no-use_srv_records"
            else:
                cmd += " --use_srv_records"
~~~

does this make sense?

Reply:
It makes perfect sense, I did originally set out to do that, but couldn't find any documentation on the `--use_srv_recrods` flag.